src/otx/algo/detection/rtdetr.py

```diff
@@ -0,0 +1,438 @@
+
+        if len(visited) < len(names):
+            unseen = set(names) - set(visited)
+            params = {k: v for k, v in model.named_parameters() if v.requires_grad and k in unseen}
+            param_groups.append({"params": params.values()})
+            visited.extend(list(params.keys()))
+
+        return param_groups
+
+    @property
+    def _exporter(self) -> OTXModelExporter:
+        """Creates OTXModelExporter object that can export the model."""
+        if self.image_size is None:
+            raise ValueError(self.image_size)
+
+        return OTXNativeModelExporter(
+            task_level_export_parameters=self._export_parameters,
+            input_size=self.image_size,
+            mean=self.mean,
+            std=self.std,
+            resize_mode="standard",
+            swap_rgb=False,
+            via_onnx=False,
+            onnx_export_configuration={
+                "input_names": ["images"],
+                "output_names": ["bboxes", "labels", "scores"],
+                "dynamic_axes": {
+                    "images": {0: "batch"},
+                    "boxes": {0: "batch", 1: "num_dets"},
+                    "labels": {0: "batch", 1: "num_dets"},
+                    "scores": {0: "batch", 1: "num_dets"},
+                },
+                "autograd_inlining": False,
+                "opset_version": 16,
+            },
+            output_names=["bboxes", "labels", "scores"],
+        )
+
+    @property
+    def _optimization_config(self) -> dict[str, Any]:
+        """PTQ config for RT-DETR."""
+        return {"model_type": "transformer"}
+
+
+class OTXRtDetr18(OTXRtDetr):
+    load_from = (
+        "https://github.com/lyuwenyu/storage/releases/download/v0.1/rtdetr_r18vd_5x_coco_objects365_from_paddle.pth"
+    )
+
+    def _build_model(self, num_classes: int) -> nn.Module:
+        backbone = PResNet(
+            depth=18,
+            pretrained=True,
+            freeze_at=-1,
+            return_idx=[1, 2, 3],
+            num_stages=4,
+        )
+        encoder = HybridEncoder(
+            in_channels=[128, 256, 512],
+            feat_strides=[8, 16, 32],
+            hidden_dim=256,
+            expansion=0.5,
+            dim_feedforward=1024,
+            eval_spatial_size=self.image_size[2:],
+        )
+        decoder = RTDETRTransformer(
+            num_classes=num_classes,
+            num_decoder_layers=3,
+            feat_channels=[256, 256, 256],
+            feat_strides=[8, 16, 32],
+            hidden_dim=256,
+            eval_spatial_size=self.image_size[2:],
+        )
+        criterion = RTDetrCriterion(
+            weight_dict={"loss_vfl": 1.0, "loss_bbox": 5, "loss_giou": 2},
+            num_classes=num_classes,
+            gamma=2.0,
+            alpha=0.75,
+        )
+        optimizer_configuration = [
+            {"params": "^(?=.*backbone)(?=.*norm).*$", "weight_decay": 0.0, "lr": 0.00001},
```

Comment:
```suggestion
            {"params": "^(?=.*backbone)(?=.*norm).*$", "weight_decay": 0.0, "lr": 0.00001},  # apply to backbone and norm
```
I'm not sure it's right comment, but how about adding comment like this? I think it can make it more readable.

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/2d5cef1ef2bbc576bebd10fd1a96e812d6ef0548